lib/phoenix_slime.ex

```diff
@@ -1,2 +1,42 @@
 defmodule PhoenixSlime do
+  @doc """
+  Provides the `~l` sigil with HTML safe Slime syntax inside source files.
+
+  Raises on attempts to interpolate with `\#{}`. Use `~L` to interpolate.
+
+      iex> import PhoenixSlime
+      iex> assigns = %{w: "world"}
+      iex> ~l"\""
+      ...> p = "hello " <> @w
+      ...> "\""
+      {:safe, [[["" | "<p>"] | "hello world"] | "</p>"]}
+  """
+  defmacro sigil_l(expr, opts) do
+    handle_sigil(expr, opts, __CALLER__.line)
+  end
+
+  @doc """
+  Provides the `~L` sigil with HTML safe Slime syntax inside source files.
+
+      iex> import PhoenixSlime
+      iex> ~L"\""
+      ...> p hello #{"world"}
```

Comment:
I believe, it should be 
```suggestion
      ...> p hello \#{"world"}
```
so interpolation is actually present in string at sigil compile time